app/models/user.rb

```diff
@@ -107,7 +107,7 @@ def validate
   def current_group_by_description=(group_description)
     if group_description
       desired_group = miq_groups.detect { |g| g.description == group_description }
-      desired_group ||= MiqGroup.find_by(:description => group_description) if super_admin_user?
+      desired_group ||= MiqGroup.in_my_region.find_by(:description => group_description) if super_admin_user?
```

Comment:
Does it make more sense for the `find_by` call to happen in the region of the user instead of just the current region?

@gtanzillo Any thoughts on this one?

Reply:
done.